docs/backup-cr.md

```diff
@@ -0,0 +1,103 @@
+# PerconaXtraDBClusterBackup Custom Resource options
+
+A Backup resource is a Kubernetes object that tells the Operator how
+to backup your database. The [deploy/backup/backup.yaml :octicons-link-external-16:](https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/backup/backup.yaml) file is a template for creating backup resources.
+
+It defines the `PerconaXtraDBClusterBackup` resource. 
+
+This document describes all available options that you can use to customize a backup. 
+
+## `apiVersion`
+
+Specifies the API version of the Custom Resource.
+`pxc.percona.com` indicates the group, and `v1` is the version of the API.
+
+## `kind`
+
+Defines the type of resource being created: `PerconaXtraDBClusterBackup`.
+
+## `metadata`
+
+The metadata part contains the following keys:
+
+* <a name="backup-metadata-name"></a> `name` sets the name of your backup resource;
+* `finalizers` subsection:
+
+    * `percona.com/delete-backup` if present, enables deletion of backup files from a backup storage when the backup object is removed (manually or by schedule). When used with the Persistent Volume as the backup storage, the finalizer deletes the PVC. 
+
+## `spec` section
+
+The toplevel spec elements of the [deploy/backup/backup.yaml :octicons-link-external-16:](https://github.com/percona/percona-xtradb-cluster-operator/blob/main/deploy/backup/backup.yaml) are the following ones:
+
+### `pxcCluster`
+
+The name of the Percona XtraDB Cluster to back up.
+
+| Value type  | Example    |
+| ----------- | ---------- |
+| :material-code-string: string     | `cluster1`         |
+
+### `storageName`
+
+The name of the storage configuration defined in your `deploy/cr.yaml` file in the `spec.backup.storages` subsection.
+
+| Value type  | Example    |
+| ----------- | ---------- |
+| :material-code-string: string     | `fs-pvc`         |
+
+### `activeDeadlineSeconds`
+
+The timeout value in seconds, after which backup job will automatically fail.
+
+| Value type  | Example    |
+| ----------- | ---------- |
+| :material-numeric-1-box: int     | `3600` |
+
+### `startingDeadlineSeconds`
+
+The maximum time in seconds for a backup to start. The Operator compares the timestamp of the backup object against the current time. If the backup is not started within the set time, the Operator automatically marks it as "failed".
```

Comment:
In the same vein with my comment about runningDeadlineSeconds, people might confuse this with `runningDeadlineSeconds`. I think we need to say:

The maximum time in seconds for a backup job to reach Starting state.